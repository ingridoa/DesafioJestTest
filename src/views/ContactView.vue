<template>
  <div class="home">
    <h1>Este es el Contacto</h1>
  </div>
</template>

<script>
export default {
  name: 'ContactView'
}
</script>
