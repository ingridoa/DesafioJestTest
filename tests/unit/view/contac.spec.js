import { mount } from "@vue/test-utils";
// import ContactView from "@/views/ContactView.vue";
import ContactView from "@/views/ContactView.vue";

describe("Test de vista About", () => {
  it("Match de snapshot con ContactView.vue", () => {
    /* Seleccionar el componente que será puesto a prueba */
    const wrapper = mount(ContactView);
    /* Assertion => Aserción */
    expect(wrapper.html()).toMatchSnapshot();
  });
});

