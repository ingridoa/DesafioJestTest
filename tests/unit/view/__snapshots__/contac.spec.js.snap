// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test de vista About Match de snapshot con ContactView.vue 1`] = `
<div class="home">
  <h1>Este es el Contacto</h1>
</div>
`;
